### DISCUSSION TITLE
MIFARE Ultralight

### DISCUSSION THREAD

**Discussion by PritorianZ** [wh201906/Proxmark3GUI#56] [General]
Hello my dear friend!
First of all, thank you so much for your work and the work you have done!
I really like your GUI, I use it very often!
Please tell me, are you planning to add support for MIFARE Ultralight?
I will be very grateful to you, this is really very useful for me.
Thank you!

**Comment by wh201906** [comment]
Actually I haven't play with MIFARE Ultralight before. I might add some features for it in the future.

**Comment by PritorianZ** [comment]
Very grateful!
